drivers/i2c/mcp23017_driver_test.go

```diff
@@ -11,18 +11,18 @@ import (
 	"gobot.io/x/gobot/gobottest"
 )
 
-var _ gobot.Driver = (*MCP23017Driver)(nil)
```

Comment:
I'd leave this line alone since it is actually a test that ensures that the driver supports the `gobot.Driver` interface, or else it will not compile.

Reply:
Thank you for reviewing the changes, I've added that line back to the test file.